UM/Qt/qml/UM/Switch.qml

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2022 UltiMaker
```

Comment:
I think this component is the best written piece of code, I have ever seen! Good job @Joeydelarago